examples/celery-rabbitmq/flower-controller.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1beta3
+kind: ReplicationController
+metadata:
+  labels:
+    name: flower
+  name: flower-controller
+spec:
+  replicas: 1
+  selector:
+    name: flower
+  template:
+    metadata:
+      labels:
+        app: taskQueue
+        name: flower
```

Comment:
It's not a blocker.

The selector in the replication controller needs to match the pods generated by the template, so if you change "name" to "component" here, it also needs to be changed under "selector".